app/main/spawn.js

```diff
@@ -51,7 +52,7 @@ export function startUserAgentService(userAgentClient, options = {}) {
   // Connecting to a client immediately is sometimes not necessary,
   // e.g. when starting this service standalone.
   if (userAgentClient) {
-    userAgentClient.connect({ port, host, version });
+    userAgentClient.connect({ port, host, version, logger });
```

Comment:
Why is the `logger` here passed as a param, instead of importing it where required?